modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+:_content-type: REFERENCE
+[id="ztp-sno-siteconfig-config-reference_{context}"]
+= {sno-caps} SiteConfig CR installation reference
+
+.SiteConfig CR installation options for {sno} clusters
+[cols="1,3", options="header"]
+|====
+|SiteConfig CR field
+|Description
+
+|`spec.cpuPartitioningMode`
+|Configure workload partitioning by setting the value for `cpuPartitioningMode` to `AllNodes`.
+To complete the configuration, specify the `isolated` and `reserved` CPUs in the `PerformanceProfile` CR.
+You enable workload partitioning during cluster installation only.
```

Comment:
cpuPartitioningMode is not GA in 4.13. This field needs to be marked as Tech Preview.

Can we reword either line 16 or 18 to make clear that enabling is done by setting cpuPartitioningMode to AllNodes. Maybe line 16 "Enable workload partitioning by setting the value...."